.github/workflows/cut-release-branch.yml

```diff
@@ -125,20 +122,33 @@ jobs:
           message="New branch ${{ inputs.new_branch_name }} created"
           echo "::notice title="New branch created": $title::$message"
 
-      # TODO: need to add scripts into the adapters to automate bumping the core 
-      # dependency on the `*.latest` branch.  The script should also bump the 
-      # nightly release test branch to the new `*.latest` branch
-      # - name: "Bump dependencies via script"
-      #   # This bumps the dependency on dbt-core in the adapters
-      #   if: ${{ !contains(github.repository, 'dbt-core') }}
-      #   run: |
-      #     echo ${{ github.repository }}
-      #     echo "running some script yet to be written now"
-      #     scripts/update_dependencies.sh
-      #     git status
-      #     git add .
-      #     git commit -m "$commit_message"
-      #     git push
+      - name: "Bump dependencies via script"
+        # This bumps the dependency on dbt-core in the adapters
+        if: ${{ !contains(github.repository, 'dbt-core') }}
+        run: |
+          echo ${{ github.repository }}
+          echo "running update_dependencies script"
+          chmod +x ${GITHUB_WORKSPACE}/.github/scripts/update_dependencies.sh
+          ${GITHUB_WORKSPACE}/.github/scripts/update_dependencies.sh ${{ inputs.new_branch_name }}
+          commit_message="bumping .latest branch variable in update_dependencies.sh to ${{ inputs.new_branch_name }}"
+          git status
+          git add .
+          git commit -m "$commit_message"
+          git push
+
+      - name: "Bump env variable via script"
+        # bumps the RELEASE_BRANCH variable in nightly-release.yml in adapters
+        if: ${{ !contains(github.repository, 'dbt-core') }}
+        run: |
+          echo ${{ github.repository }}
+          echo "running some script yet to be written now"
+          chmod +x ${GITHUB_WORKSPACE}/.github/scripts/update_release_branch.sh
```

Comment:
nit: I don't think we need to chmod this every time (unless we're creating this file at runtime somehow)

Reply:
without the +x command was showing lack of permissions since the scripts are in a different repo technically so not sure if there is a way to just permanently give them or not.

Reply:
It's odd that `chmod` solves the cross repo thing. I suspect there's a different permission error. What's the error that you get if you don't use `chmod`?

Reply:
Does this work?

```bash
run: |
  echo ${{ github.repository }}
  echo "running some script yet to be written now"
  bash ${GITHUB_WORKSPACE}/.github/scripts/update_release_branch.sh ${{ 
inputs.new_branch_name }}
  commit_message="updating env variable to ${{ inputs.new_branch_name }} in nightly-release.yml"
  git status
  git add .
  git commit -m "$commit_message"
  git push
```

Reply:
this does infact work thanks! i think i had a typo in path when i originally tried it this way so was getting unexpected error change made